nginx/Dockerfile

```diff
@@ -14,10 +14,11 @@
 # limitations under the License.
 
 # https://github.com/kubernetes/ingress-nginx
-FROM k8s.gcr.io/ingress-nginx/controller:v1.1.2
+FROM k8s.gcr.io/ingress-nginx/controller:v0.51.0
```

Comment:
downgrade?

Reply:
@andriisoldatenko we need to release both an 0.x and 1.x image. We should find a way to improve this process, but for now this is a simple workaround.